pkpdapp/pkpdapp/tests/test_views.py

```diff
@@ -69,6 +71,115 @@ def test_index_contains_correct_html(self):
         self.assertContains(response, contents)
 
 
+class TestDatasetView(TestCase):
+    """
+    Tests the dataset view.
+    """
+    def setUp(self):
+        self.test_dataset = Dataset.objects.create(
+            name='my_cool_dataset',
+            datetime=timezone.now(),
+            description='description for my cool dataset',
+            administration_type='T1',
+        )
+
+    def test_view_uses_correct_template(self):
+        response = self.client.get('/dataset/{}/'.format(self.test_dataset.id))
+        self.assertEquals(response.status_code, 200)
+        self.assertTemplateUsed(response, 'dataset_detail.html')
+
+    def test_index_contains_correct_html(self):
+        response = self.client.get('/dataset/{}/'.format(self.test_dataset.id))
+        self.assertContains(response, self.test_dataset.name)
+        self.assertContains(response, self.test_dataset.description)
+
+
+class TestPkpdModelView(TestCase):
```

Comment:
I'm wondering whether PKPD model is the best naming (as discussed splitting into PK, PD and potentially Disease Model makes more sense, unless the PKPD model would reference to those models and parse them together for this particular project).

For the moment I think it's good though, but when we build the infrastructure we probably want to keep that in mind.

Reply:
agree its probably not the final name, but as its currently written the model can represent both PK and PD models, so fits with current functionality